libraries/vendor/composer/autoload_classmap.php

```diff
@@ -6,10 +6,772 @@
 $baseDir = dirname(dirname($vendorDir));
 
 return array(
+    'Cache_Lite' => $vendorDir . '/pear/cache_lite/Cache/Lite.php',
```

Comment:
We still have to get the CMS there.  When we merged Composer resources at 3.4 we decided to commit production code in the vendor folder.  There were two arguments for this; enabling contributors to clone the repo without needing to use extra tools too (Composer) and packaging.  Our patch packages are built by comparing git tags and if the folder isn't tracked in VCS then an alternate approach is needed to decide what to include from vendor.  FWIW in Mautic we use a similar packaging technique and basically take an all-or-nothing approach to it (if .lock is changed include the whole folder).